### DISCUSSION TITLE
Installer Copyright is 2022 and not 2023

### DISCUSSION THREAD

**Discussion by A-Loot** [valinet/ExplorerPatcher#1115] [General]
This isn't really anything important, but when you open the Properties/Details page for the Installer it says Copyright 2006-2022 instead of 2006-2023.

Also, for some reason both Chrome and Windows think that the installer for 22621.1413.54.1 isn't safe, even though the installers for previous versions are.

**Comment by valinet** [comment]
I will update that in the next version, thanks.

It's a practice these days to flag indie unsigned applications as "malicious", to distract people from doing what they want with their machines, and instead sticking only to programs promoted by various app stores and so on. ExplorerPatcher has its entire source code published, it is built on untouched machines in Microsoft's cloud and the binaries you get are untouched, straight what those computers have compiled (you can check the contents against the artifacts produced, you will see they are identical). By the nature of such an app, ExplorerPatcher employs some behaviors that might also be _usually_ found in real malware applications: the tools are there, offered by the OS, it is up to the developer on what he/she decides to do with them. ExplorerPatcher doesn't do anything malicious to that end, using all these powers only to achieve its goal, and the libre nature when it comes to its auditing and lack of telemetry I think speak for themselves. Thanks.

**Comment by A-Loot** [comment]
Weirdly the copyright is still the same in the newest version.

![image](https://user-images.githubusercontent.com/105282948/226141154-193b607a-facf-4309-aa99-1000ad3d95ae.png)

**Reply by valinet** [reply]
Yeah, fml. Re-released the update now, it's fixed.

**Comment by Anixx** [comment]
It is not necessary to maintain copyright later date. Many products on the market are sold with such outdated copyright notice. What is more importtant in the starting date.

**Reply by valinet** [reply]
Doesn't really matter, to be honest.
